src/app/features/attendance/attendance-week-dashboard/attendance-week-dashboard.component.ts

```
@@ -109,12 +104,12 @@ export class AttendanceWeekDashboardComponent
 
       rows
         .filter((r) => this.filterLowAttendance(r))
-        .forEach((r) => lowAttendanceCases.add(r.childId));
+        .forEach((r) => lowAttendanceCases.add(r.participantId));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)